examples/gce-machinedeployment.yaml

```diff
@@ -49,6 +49,10 @@ spec:
             machineType: "n1-standard-2"
             # In GB
             diskSize: 50
+            # ether of two network/subnetwork could be specified
+            # in case if both empty — default network will be used
+            network: "my-cool-network"
```

Comment:
Maybe just a dumb question from my side, not yet took a deeper look into network in GCE. Are there naming conventions?

Reply:
names could be ether "name" or selflink. both works.